core/sql/ustat/hs_cli.cpp

```diff
@@ -524,12 +531,14 @@ Lng32 HSSample::create(NAString& tblName, NABoolean unpartitioned, NABoolean isP
            tableOptions = " WITH PARTITIONS";
         // If a transaction is running, the table needs to be created as audited.
         // Otherwise, create table as non-audited.
+        /* TEMPTEMP. Dave need to validate this change.
         if (TM->InTransaction())
           tableOptions += " ATTRIBUTE AUDIT";
         else
           tableOptions += " ATTRIBUTE NO AUDIT";
 
         tableOptions += getTempTablePartitionInfo(unpartitioned, isPersSample);
+        */
```

Comment:
Commenting out the "ATTRIBUTE [NO] AUDIT" stuff is fine. (Indeed it appears that Trafodion does not support non-audited tables. I tried creating one and DMLs on it fail with error 4211.) But the getTempTablePartitionInfo method supplies a SALT clause, which is important to get a nice degree of parallelism when we populate the sample table. So I think we want to keep that.